Conceptpower+Spring/src/main/java/edu/asu/conceptpower/web/ConceptWrapperAddController.java

```diff
@@ -83,9 +84,13 @@ private void initBinder(WebDataBinder binder) {
      */
     @RequestMapping(value = "auth/conceptlist/addconceptwrapper")
     public String prepareConceptWrapperAdd(@ModelAttribute ConceptWrapperAddBean conceptWrapperAddBean,
```

Comment:
since you only need to pass the id, you don't need to inject the whole object here. just use the request parameter annotation.

Reply:
Changed. Passed only the id instead of the whole object.